scoap3/modules/tools/tools.py

```diff
@@ -39,74 +37,69 @@ def affiliations_export(country=None, year=None):
     result_headers = ['year', 'journal', 'doi', 'arxiv number', 'primary arxiv category',
                       'country', 'affiliation', 'authors with affiliation', 'total number of authors']
     result_data = []
-    index = 0
 
     # query ElasticSearch for result (and get total hits)
     query = get_query_string(country=country, year=year)
     search_results = current_search_client.search(
-        q=query, index=search_index, _source=source_fields, size=size, from_=index
-    )
+        q=query, index=search_index, _source=source_fields, scroll='1m')
+
+    sid = search_results['_scroll_id']
+    scroll_size = len(search_results['hits']['hits'])
```

Comment:
```suggestion
    scroll_size = results['hits']['total']['value']
```

Reply:
But the scroll size has to be the same size as the page size or smaller

Reply:
<img width="390" alt="Screenshot 2023-01-26 at 11 28 02" src="https://user-images.githubusercontent.com/36658177/214813708-277998cc-7656-411f-92cb-227b7f0d7338.png">

Reply:
true!